config.toml

```diff
@@ -4,7 +4,7 @@ relativeurls = true
 canonifyURLs = false
 languageCode = "en-us"
 title = "Live Free or Dichotomize"
-theme = "tuftesque"
+theme = "tuftesqueGit"
```

Comment:
why is it called tuftesqueGit instead of just tuftesque? Like if someone wanted to use it would they have to call it that?